config/bidderinfo.go

```diff
@@ -276,6 +276,7 @@ func processBidderAliases(aliasNillableFieldsByBidder map[string]aliasNillableFi
 		if err := validateAliases(aliasBidderInfo, bidderInfos, bidderName); err != nil {
 			return nil, err
 		}
+		openrtb_ext.SetAliasBidderName(openrtb_ext.BidderName(string(bidderName))) //required for CoreBidderNames function to also return aliasBiddernamess
```

Comment:
Nitpick: please put a comment on top of the target line, it will make it easier to read

Reply:
thank you fixed here - https://github.com/prebid/prebid-server/pull/3027/commits/4c624f50f0c9f2b04d1a1cec74f88ddf47998eb6